sklearn/datasets/_base.py

```diff
@@ -687,33 +691,32 @@ def load_breast_cancer(*, return_X_y=False, as_frame=False):
         data : {ndarray, dataframe} of shape (569, 30)
             The data matrix. If `as_frame=True`, `data` will be a pandas
             DataFrame.
-        target: {ndarray, Series} of shape (569,)
+        target : {ndarray, Series} of shape (569,)
             The classification target. If `as_frame=True`, `target` will be
             a pandas Series.
-        feature_names: list
+        feature_names : list
             The names of the dataset columns.
-        target_names: list
+        target_names : list
             The names of target classes.
-        frame: DataFrame of shape (569, 31)
+        frame : DataFrame of shape (569, 31)
             Only present when `as_frame=True`. DataFrame with `data` and
             `target`.
 
             .. versionadded:: 0.23
-        DESCR: str
+        DESCR : str
             The full description of the dataset.
-        filename: str
+        filename : str
             The path to the location of the data.
 
             .. versionadded:: 0.20
 
     (data, target) : tuple if ``return_X_y`` is True
+        A tuple of two ndarrays. The first contains a 2D array of shape (569, 30)
+        with each row representing one sample and each column representing the features.
+        The second array of shape (569,) contains the target samples.
```

Comment:
```suggestion
        A tuple of two ndarrays by default. The first contains a 2D ndarray of
        shape (569, 30) with each row representing one sample and each column
        representing the features. The second ndarray of shape (569,) contains
        the target samples.  If `as_frame=True`, both arrays are pandas objects.
```
Maybe being more explicit here could make the description easier to understand. 

Also try to follow the PEP 8 convention, i.e. limit all lines to a maximum of 79 characters and avoid blank spaces at the end of each line.

Reply:
@ArturoAmorQ I followed the suggestions from #22266 and got the tests to pass.

Reply:
You should check the suggestion of @ArturoAmorQ that proposes an improvement regarding the writing.

Reply:
@glemaitre  but the pytest tests on my local computer already passed.

Reply:
It is not about the testing here but about the English writing.
Check the proposal above: https://github.com/scikit-learn/scikit-learn/pull/22346/files#r795568092

Reply:
@glemaitre I think there's a bit of  confusion. What I'm trying to say is that i ran pytest sklearn/tests/test_docstrings.py -k sklearn.datasets._base.load_breast_cancer in my local computer and the tests passed 100%. Also I would like to know why this branch isn't merged.

Reply:
Just to be explicit, I was referring to the change present in this commit: https://github.com/scikit-learn/scikit-learn/pull/22346/commits/3603e53ab5b21ca84feeac14f7748a379b0d4214